src/components/JoinPage/index.js

```diff
@@ -7,28 +7,41 @@ import Form from 'react-bootstrap/Form';
 import CreateButton from '../CreateButton';
 import JoinButton from '../JoinButton';
 
+import './styles.css';
+
 function JoinPage() {
     const [name, setName] = useState('');
     const [room, setRoom] = useState('');
 
     return (
         <Container fluid>
-            <Row></Row>
             <Row>
                 <Col></Col>
-                <Col>
-                    <Form onSubmit={e => e.preventDefault()}>
-                        <Form.Group controlId="formName">
-                            <Form.Label>Name</Form.Label>
-                            <Form.Control required type="text" placeholder="Enter name" onChange={e => setName(e.target.value)} />
-                        </Form.Group>
-                        <CreateButton name={name} />
-                        <JoinButton name={name} room={room} setRoom={setRoom} />
-                    </Form>
+                <Col xs={6}>
+                    <div className="cardHeader">
+                        <div className="card-body">
+                            <img src="../../assets/myWord_Logo-01.png" alt="logo" className="logo" href="logo"/>
+                        </div>
+                    </div>
+                    <div className="card">
```

Comment:
if this is a bootstrap card, we should use the React Bootstrap version instead:
https://react-bootstrap.github.io/components/cards/

Dig through the docs for any bootstrap components you want. The React Bootstrap versions will work better here and that will allow us to only use the npm version of bootstrap (which we need as a dependency of react-bootstrap)